src/pages/TeamSettings/KV.vue

```diff
@@ -0,0 +1,636 @@
+<script>
+import JsonInput from '@/components/CustomInputs/JsonInput'
+import ManagementLayout from '@/layouts/ManagementLayout'
+import ConfirmDialog from '@/components/ConfirmDialog'
+import Alert from '@/components/Alert'
+// import ExternalLink from '@/components/ExternalLink'
+
+import MenuTooltip from '@/components/MenuTooltip'
+import { formatTime } from '@/mixins/formatTimeMixin'
+import { mapGetters } from 'vuex'
+
+export default {
+  components: {
+    JsonInput,
+    Alert,
+    ManagementLayout,
+    ConfirmDialog,
+    MenuTooltip
+  },
+  mixins: [formatTime],
+  data() {
+    return {
+      // loading states
+      isFetchingKV: true,
+      isDeletingKV: false,
+      isSettingKV: false,
+
+      // Store previous kv name when modifying kv
+      // This is used to delete the kv with that name and create a new kv with a separate value
+      previousKVName: null,
+
+      // KV selected for modification/deletion
+      selectedKV: null,
+
+      // Alert data
+      alertShow: false,
+      alertMessage: '',
+      alertType: null,
+
+      // Input rules
+      rules: {
+        required: val => !!val || 'This field is required.'
+      },
+      errorMessage: '',
+      invalidKV: false,
+
+      // Types
+      selectedTypeIndex: 0,
+      kvTypes: [
+        { value: 'auto', text: 'Auto' },
+        { value: 'string', text: 'String' },
+        { value: 'json', text: 'JSON' }
+      ],
+
+      // JsonInput
+      placeholderText:
+        "Enter your KV value here...\n\nClick on 'Type' to select a type validation",
+
+      // Create/modify kv key & value input
+      keyInput: null,
+      KvValueInput: '',
+
+      // Distinguish between creating & modifying KV
+      isKvUpdate: false,
+
+      jsonInput: '',
+
+      // table search
+      search: '',
+
+      expanded: [],
+
+      //table headers
+      headers: [
+        { text: 'Key', value: 'key' },
+        { text: 'Value', value: 'value' },
+        { text: 'Created By', value: 'tenant.name' },
+        { text: 'Created', value: 'created' },
+        { text: 'Last Updated', value: 'updated' },
+        {
+          text: '',
+          value: 'actions',
+          align: 'right',
+          sortable: false
+        }
+      ],
+
+      items: [],
+
+      // Dialogs
+      keyModifyDialog: false,
+      keyDeleteDialog: false
+    }
+  },
+  computed: {
+    ...mapGetters('tenant', ['tenant', 'role']),
+    disableConfirm() {
+      if (!this.keyInput) return false
+      if (!this.KvValueInput) return false
+      if (this.invalidKV) return false
+      return true
+    },
+    isReadOnlyUser() {
+      return this.role === 'READ_ONLY_USER'
+    }
+  },
+  watch: {
+    tenant() {
+      this.$apollo.queries.kv.refetch()
+    }
+  },
+  methods: {
+    checkValue(value) {
+      return typeof value == 'object' ? JSON.stringify(value) : value
```

Comment:
I get an error if I enter a number as a value. CodeMirror expects it to be input as a string.  Is there a reason we're only stringify-ing an object?